cli/script/command-executor.ts

```diff
@@ -1223,6 +1364,43 @@ function requestAccessToken(): Promise<string> {
     });
 }
 
+export var runReactNativeBundleCommand = (entryFile: string, outputFolder: string, platform: string, sourcemapOutput: string): Promise<void> => {
+    var reactNativeBundleCommandArgs = [
```

Comment:
I wonder if we should use the locally installed packager to run this, as opposed to relying on the global CLI. That way we know which versions of RN we support since we'd be depending on a Node module that's shipped with the platform as opposed to a globally installed CLI. It seems a little cleaner to use a n NPM module instead of shelling to a CLI if possible as well.

https://github.com/facebook/react-native/blob/master/packager/react-packager/index.js